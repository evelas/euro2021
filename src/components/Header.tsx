import React from 'react'


const Header: React.FC = () => {
  return (
    <header className="header">
      <h1 className="header__title">UEFA EURO 2020</h1>
      <h2 className="header__about">Система управления данными</h2>
    </header>
  )
}

export default Header
