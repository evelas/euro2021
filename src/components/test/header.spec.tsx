import { shallow, ShallowWrapper, render } from "enzyme";
import React from "react";
import Header from "../Header";

const setUp = () => shallow(<Header />);

describe("Header component", () => {
  let component: ShallowWrapper;
  beforeEach(() => {
    component = setUp();
  });

  it("should contain .header wrapper", () => {
    const wrapper = component.find(".header");
    expect(wrapper.length).toBe(1);
  });
  it("should render Header component", () => {
    const component = render(<Header />);
    expect(component).toMatchSnapshot();
  });
});
