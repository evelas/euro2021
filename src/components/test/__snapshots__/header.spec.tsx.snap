// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render Header component 1`] = `
<header
  class="header"
>
  <h1
    class="header__title"
  >
    UEFA EURO 2020
  </h1>
  <h2
    class="header__about"
  >
    Система управления данными
  </h2>
</header>
`;
